.github/workflows/reusable-build-iso.yml

```diff
@@ -179,38 +179,34 @@ jobs:
           compression-level: 0
           overwrite: true
 
-      - name: Determine R2 Secrets
-        id: r2-secrets
+      - name: Upload ISOs and Checksum to R2 to Bluefin Bucket
```

Comment:
You can have this as one step by using GitHub Environments.
You can have a secret for "R2_ACCESS_KEY_ID" for Bluefin, and a separate value for Aurora.

Reply:
Hi @p5, that sounds like a good idea! So each environment has it's own secrets and you pick that in the workflow?

Reply:
Exactly.  You select which environment you wish to use at the job level, and you store separate secrets for each environment.
https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#using-an-environment

You can also have repo-level secrets which are accessible from all environments

Reply:
Sounds good. I think we should open an issue to do that and move the secrets to the correct environment. I don't have time to do that at the moment, is this PR OK with folks just to get ISOs building?